paradise_papers_django/search/templates/search/display_information/entity.html

```diff
@@ -57,110 +57,109 @@ <h1 class="company-title">{{ node_info.name }}</h1>
                <span>Agent:</span> <strong>{{ node_info.service_provider }}</strong>
             </li>
             <li>
-               <span>{{ node_info.valid_until }} </span>
+               <span>{{ node_info.valid_until }}</span>
             </li>
          </ul>
       </div>
    </div>
    <div class="search_results_cont connections">
       <h2>Connections</h2>
       {% if entity_connections %}
-
-      <h3>Entity</h3>
-      <table class="display-relationship body-table-style entity-table">
-         <tbody>
-            <thead>
-               <th></th>
-               <th class="Role">Role</th>
-               <th class="Incorporation">Incorporation</th>
-               <th class="Jurisdiction">Jurisdiction</th>
-               <th class="Status">Status</th>
-               <th class="source">Data From</th>
-            </thead>
-            {% for entity in entity_connections %}
-            <tr>
-                <td class="description"><a class="node-link" title="{{ entity.1.name }}" href="/search/{{ entity.1.node_id }}">{{ entity.1.name }}</a></td>
-                <td class="Role">{{ entity.0 | changeUnderline  }}</td>
-                <td class="Incorporation"><span>{{entity.1.incorporation_date}}</span></td>
-                <td class="Jurisdiction"><span>{{entity.1.jurisdiction}}</span></td>
-                <td class="status"><span>{{entity.1.status}}</span></td>
-                <td class="source"><a class="node-link" href="https://panamapapers.icij.org">{{entity.1.sourceID}}</a></td>
-            </tr>
-            {% endfor %}
-         </tbody>
-      </table>
+          <h3>Entity</h3>
+          <table class="display-relationship body-table-style entity-table">
+             <tbody>
+                <thead>
+                   <th></th>
+                   <th class="Role">Role</th>
+                   <th class="Incorporation">Incorporation</th>
+                   <th class="Jurisdiction">Jurisdiction</th>
+                   <th class="Status">Status</th>
+                   <th class="source">Data From</th>
+                </thead>
+                {% for entity in entity_connections %}
+                    <tr>
+                        <td class="description"><a class="node-link" title="{{ entity.1.name }}" href="/search/entity/{{ entity.1.node_id }}">{{ entity.1.name }}</a></td>
```

Comment:
The url need to be dynamic but this can be update when we refactor the way we handle the templates